.github/workflows/scorecards.yml

```diff
@@ -0,0 +1,69 @@
+name: Scorecard supply-chain security
```

Comment:
Nit: Please match existing naming conventions for our workflows. That is:

* Workflow name: `Title Case`. Ideally very brief - e.g. just `Scorecard` would be good.
* Job names: `hyphen-separated-lower-case`. Basically we omit the `name` field for jobs.
* Step names: `Title Case`. These can be longer, e.g. `Upload Artifacts to GitHub`.

See https://github.com/crossplane/crossplane/blob/master/.github/workflows/ci.yml for example.